data/keystore/build.gradle

```diff
@@ -0,0 +1,21 @@
+jar {
```

Comment:
Do we need to specify the jar properties at all? Looks like this isn't need from what other modules are doing.

Reply:
removed